templates/frontend/pages/aboutEditorialTeamBio.tpl

```diff
@@ -22,9 +22,18 @@
 	</h2>
 
 	<div class="editorialTeamBioContent">
-		{$editor->getLocalizedData('biography')|strip_unsafe_html}
+		{if $profileImage}
+			<img src="{$baseUrl}/{$publicfiles}/{$profileImageUpload}?{""|uniqid}" alt="{translate key="user.profile.form.profileImage"}" />
+		{/if}
+		{$templateEditor->getLocalizedData('biography')|strip_unsafe_html}<br>
 	</div>
 
+	<div style="display:flex;" class="orcidContent">
+		{if {$templateEditor->getData('orcid')} != null}
+			<div style="width:24px; height:24px;">{$orcidIcon}</div>
+			<a style="font-size:24px;"href = {$templateEditor->getData('orcid')}> {$templateEditor->getData('orcid')} </a>
```

Comment:
The `href` attribute is abutted to the closing quote of the `style` attribute, and is missing any quotes of its own.